myproject/blog/templates/blog/blogdetailpage.html

```diff
@@ -5,13 +5,25 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Blog Description Page</title>
-    <style>
-        
-    </style>
+    
 </head>
 <body>
-    <h1>{{ object.title }}</h1>
-    <p>{{ object.content }}</p>
+    <h1>{{ context.title }}</h1>  
+    <p><span>Posted on :</span>{{context.posted_date}}</p>
+    <p>{{ context.content }}</p>
+    <p><span>Written by :</span>{{context.author.name}}</p>
+
+    <p>Comments : </p>
+    {% for com in object_list %}
+    <ul>
+        <li>
+            {{com.comment_content}}
+            {{com.commented_date}}
+        </li>
+    </ul>
+    {% endfor %}
+    
+    <a href="http://localhost:8000/admin/blog/comment/add/">Click here to add Comments</a>
```

Comment:
Change this and use django `{% url %}`

Reply:
it will be changed in very next task as it is connected